jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/util/NotificationUtils.java

```diff
@@ -58,6 +59,7 @@ public static void triggerNotificationsForViolations(List<Annotation> annotation
         }
         catch (Exception e) {
             LOGGER.error("Error triggering lambda function url, notification request not sent. Error - {}",e.getMessage());
+            LOGGER.error("Exception occurred in job with policyUUID: "+annotations.get(0).get(POLICY_NAME)+" Error message - Error triggering lambda function url, notification request not sent, "+e.getMessage());
```

Comment:
how it this different from log message above?

Reply:
okay will merge two into one.